scripts/scrape_icons.py

```diff
@@ -0,0 +1,99 @@
+import os
+import subprocess
+
+import requests
+from bs4 import BeautifulSoup
+
+
+def get_domain_htmls(emails: str):
+    for email in emails.splitlines():
+        name, domain = email.split("@")
+        try:
+            r = requests.get(f"https://{domain}/")
+            assert r.ok, r.status_code
+            yield domain, r.content
+        except (AssertionError, requests.RequestException) as e:
+            print(f"failed to fetch domain {domain} - {e}")
+            continue
+
+
+def get_domain_logos(content: str):
+    soup = BeautifulSoup(content, "html.parser")
+    for elem in soup.find_all():
```

Comment:
Think we should limit to `img` tags?


```suggestion
    for elem in soup.find_all("img"):
```

Reply:
wanted to also check `meta favicon` tags, but i think you're right - all of what i found was from `img` tags